code/game/objects/structures/crates_lockers/closets/secure/freezer.dm

```diff
@@ -1,5 +1,13 @@
 /obj/structure/closet/secure_closet/freezer
 	icon_state = "freezer"
+	var/jones = FALSE
+
+/obj/structure/closet/secure_closet/freezer/ex_act()
+	if(!jones)
+		jones = TRUE
+		return 0

```

Comment:
remove the return

Reply:
this needs to return 0/FALSE though for check purposes

Reply:
this return is pointless for the code right now, nothing else will trigger and it will already return `.` (which is null by default) when the proc ends